dts/bindings/dsa/dsa.yaml

```
@@ -4,7 +4,9 @@
 description: DSA Device
 
 child-binding:
-  description: Properties of slave port
+  description: Properties of port
+
+  compatible: "dsa-port"
```

Won't this throw warnings of an unknown/missing vendor?

```suggestion
  compatible: "zephyr,dsa-port"
```